### DISCUSSION TITLE
GitHub Action Skipping the Deploy Section

### DISCUSSION THREAD

**Discussion by callistusndemo** [jpanther/congo#162] [Q&A] [answered]
I've been trying to set up a new Hugo site using Congo as the theme. I followed the [GitHub Pages Hosting and Deployment](https://jpanther.github.io/congo/docs/hosting-deployment/#github-pages) instruction in the Congo docs but the site is not being published. I've ran it locally and it works well. When I checked the GH Action logs, for some reason the Deploy part is being skipped. Or so I assume. See below.  

![image](https://user-images.githubusercontent.com/29136312/160966581-2fe8c73c-4ea4-402b-b1d5-3352ebcb5f94.png)

One odd observation thing, when I go to the **Pages > Settings** to change the deploy branch to `gh-pages`, it's not listed in the dropdown. Not sure why. 

![image](https://user-images.githubusercontent.com/29136312/160967682-bb6a0f9b-af60-4941-b7b7-dd726b7b6569.png)

Different attempts to fix the issue have not really worked. Would greatly appreciate any pointers on what I might be missing. Thank you! 

* [Blog Repository](https://github.com/callistusndemo/callistusndemo.github.io)
* [GitHub Action YAML file](https://github.com/callistusndemo/callistusndemo.github.io/blob/master/.github/workflows/gh-pages.yml)

**Comment by jpanther** [comment, marked as answer]
I had a quick look at your repo and it may be that you need to change `${{ github.ref == 'refs/heads/main' }}` in the Deploy action to `${{ github.ref == 'refs/heads/master' }}` as your default branch is named `master`.

**Reply by callistusndemo** [reply]
THANK YOU! 🙇🏾
